### PR TITLE
OCPBUGS-35209: Update centos image and repo for openshift ansible

### PR DESCRIPTION
**Pull request by barbacbd**
** All of the references were to centos8 which is end of life. Alter these images to use a centos9 reference.

### CODE REVIEW

File: ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 base_rpm_images:
   os:
-    name: centos
+    name: centos-stream-coreos-9
```
Comment (deepsm007) [inline ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml:3]: name: centos
    namespace: openshift
    tag: stream9

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.13-e2e-aws-workers-rhel8
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.13-images

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.13-images

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.17-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.14-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.15-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.15-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.16-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.14-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
/pj-rehearse pull-ci-openshift-openshift-ansible-release-4.17-e2e-aws-workers-rhel8

**Comment by barbacbd** [comment]
pj-rehearse ack

**Review by r4f4 — COMMENTED** [review]
/lgtm

**Pull request merged**